README.md

```diff
@@ -108,13 +110,24 @@ tokenizer = AutoTokenizer.from_pretrained('bert-base-cased')
 tokenized_dataset = squad_dataset.map(lambda x: tokenizer(x['context']), batched=True)
 ```
 
+If your dataset is bigger than your disk or if you don't want to wait to download the data, you can use streaming:
+
+```python
+# If you want to efficiently download the data as you iterate over the dataset
```

Comment:
Again here, "download" can be understood as "save to disk".

Reply:
Changed to

> # If you want to use the dataset immediately and efficiently stream the data as you iterate over the dataset